.gitignore

```diff
@@ -1,3 +1,2 @@
 /.tscache
-/dist/
```

Comment:
we do not version dist, it is not required and only pollutes diffs. I generate dist in release commits only.